### PR TITLE
A few python package updates

### PR DESCRIPTION
**Pull request by joachifm**
Minor updates to python expressions I maintain.

### CODE REVIEW

**Comment by Mic92** [comment]
```
@GrahamcOfBorg build ntfy octoprint python27Packages.emoji python27Packages.jsonrpclib python27Packages.ntfy python36Packages.emoji python36Packages.ntfy
```

**Comment by Mic92** [comment]
Has macOS a conditional dependency?

```
Collecting appdirs>=1.4.0 (from OctoPrint==1.3.8)
  Could not find a version that satisfies the requirement appdirs>=1.4.0 (from OctoPrint==1.3.8) (from versions: )
```

**Comment by joachifm** [comment]
No idea about octoprint, but this looks like an existing problem with the macos build: https://hydra.nixos.org/build/82721369

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2835,26 +2835,7 @@ in {
...
-    propagatedBuildInputs = with self; [ cjson ];
-
-    meta = {
-      description = "JSON RPC client library";
-      homepage = https://pypi.python.org/pypi/jsonrpclib/;
-      license = stdenv.lib.licenses.asl20;
-      maintainers = [ stdenv.lib.maintainers.joachifm ];
-    };
-  };
+  jsonrpclib = callPackage ../development/python-modules/jsonrpclib { };
```
Comment (bhipple) [inline pkgs/top-level/python-packages.nix]: Thanks for moving this out of `python-packages.nix` :+1:

**Comment by joachifm** [comment]
@abbradar is listed as the octoprint maintainer, perhaps they know whether it should be disabled on macos.

**Comment by joachifm** [comment]
Merging this, as the octoprint issue seems strictly unrelated.

**Pull request merged**